apps/scan/frontend/src/utils/get_page_count.ts

```diff
@@ -4,6 +4,16 @@ import { Buffer } from 'buffer';
 import 'pdfjs-dist/build/pdf.worker.entry';
 
 export async function getPageCount(element: JSX.Element): Promise<number> {
+  // mock for tests
+  if (process.env.NODE_ENV === 'test') {
+    return Promise.resolve(1);
+  }
+
+  // mock if doing development in the browser
+  if (!window.kiosk) {
+    return Promise.resolve(1);
+  }
+
```

Comment:
The way that we get the page count here is a little overdone - we reprint the report as a PDF through `kiosk-browser` and use `pdfjs` to get a page count. It doesn't work in tests because `kiosk.printToPDF` doesn't work. I've just added mocks so I can use this in the browser or tests without hanging on this.

Reply:
I'm OK leaving it for now - and leaving a number of other convoluted aspects of this PR - to be resolved by eventual printing from the backend. 

Part of the whole challenge here is setting up the frontend with all the information it needs to print the report, while the backend already has it.

Reply:
Sounds good, thanks for the context